### ISSUE TITLE
Reaching out to ITP learners during the course

### ISSUE THREAD

**Issue by kfklein15** [CodeYourFuture/Ops-Programme#252]
- Not done any steps
- Done step 3 incentivise
Ticket: https://github.com/CodeYourFuture/Module-Data-Flows/issues/51

**Comment by kfklein15** [comment]
https://docs.google.com/spreadsheets/d/1xNSlo5AB51hIgdfLiOzkOo7qolT1SYW3FyS37b-bmUU/edit?gid=0#gid=0

**Comment by AzinYad** [comment]
SO I have managed to talk to 3 more of our ITP learners @kfklein15 , one asked to be moved to May, One was abit confused(Sheffield ) and the other one is ready to submit in 1/2 days

So now we need to send SMS only to two people, one based in the UK aand the other ones number is from Europe

**Comment by kfklein15** [comment]
Aamzing work.

The person from Europe. Are they on Slack at all? Have they signed Docuseal?

I am not sure we need to send SMS to one person. An email should be enough :D 

Sheffield confused: what do you mean?
Did you move the other person to May?

**Comment by AzinYad** [comment]
Yes I moved Sogand (Sue) to May and send her a confirmation email about that,
Mohammed Alenizy from Sheffield hasn’t been to any classes since the start of ITP he wasn’t sure what he needs to submit which I Explained.
for Maximilian Rendek, I couldn’t find any agreements on Docuseal.

**Comment by AzinYad** [comment]
@kfklein15 can I move it to backlog, as we have long way till Step 3 Submission ?

**Comment by kfklein15** [comment]
Cool, let's archive Maximilian then, I doubt he is actually on ITP
I think you can close this ticket.
